.github/workflows/test.yml

```diff
@@ -821,7 +820,6 @@ jobs:
             './tfm/cypress/e2e/journeys/case-amendments/amendment-underwriting-page-multiple-amendments/*.spec.js',
             './tfm/cypress/e2e/journeys/case-amendments/amendments-version-order/*.spec.js',
             './tfm/cypress/e2e/journeys/case-amendments/facility-end-date/bss-deal-expect-no-facility-end-date-features/*.spec.js',
-            './tfm/cypress/e2e/journeys/case-amendments/facility-end-date/feature-flag-disabled/*.spec.js',
```

Comment:
Looks like this should remain, to cover this test?

```ts
e2e-tests/tfm/cypress/e2e/journeys/case-amendments/facility-end-date/feature-flag-disabled/gef-deal-expect-no-facility-end-date-features.spec.js
```

Reply:
That is a feature-flag-disabled test & will fail if the feature flag is on (now I've removed the if statement checking this)

Reply:
Ah sorry, didn't realise this is in the "FF enabled" suite